masterdata/configuration/locations/locations.csv

```diff
@@ -1,19 +1,12 @@
-Uuid,Void/Retire,Name,Description,Parent,Attribute|id_prefix,Tags,Address 1,Address 2,Address 3,Address 4,Address 5,Address 6,City/Village,County/District,State/Province,Postal Code,Country
-72636eba-29bf-4d6c-97c4-4b04d87a95b5,,Bahmni Hospital,,,,Facility,,,,,,,,,,,
-1973e910-160e-42ef-9daf-3f020f62c4cf,,Registration Desk,,Bahmni Hospital,,Login Location; Visit Location,,,,,,,,,,,
-5e232c47-8ff5-4c5c-8057-7e39a64fefa5,,OPD,,Bahmni Hospital,,Login Location; Visit Location; Appointment Location,,,,,,,,,,,
-b5da9afd-b29a-4cbf-91c9-ccf2aa5f799e,,Emergency,,Bahmni Hospital,,Login Location; Visit Location; Admission Location,,,,,,,,,,,
-8e6ad830-21be-488b-87af-5e480334342c,,General Ward,,Bahmni Hospital,,Login Location; Visit Location; Admission Location,,,,,,,,,,,
-046a84ac-cd6a-4725-b712-24cb115d1c55,,General Ward Room-1,,General Ward,,Admission Location,,,,,,,,,,,
-6f9ad5d9-bcfe-45fb-82b1-c48bf6ae051f,,General Ward Room-2,,General Ward,,Admission Location,,,,,,,,,,,
-815ddfb1-5b53-4fc8-a53a-3b579d4049b0,,Labour Ward,,Bahmni Hospital,,Login Location; Visit Location; Admission Location,,,,,,,,,,,
-fdb410d7-63ec-486b-bab8-1e1c990998cb,,Labour Ward Room-1,,Labour Ward,,Admission Location,,,,,,,,,,,
-5f42470d-5208-4b79-900e-2ffa06aebb88,,Labour Ward Room-2,,Labour Ward,,Admission Location,,,,,,,,,,,
-ade547cd-49b3-4025-b9e3-f2e50126028f,,ANC Ward,,Bahmni Hospital,,Login Location; Visit Location; Admission Location,,,,,,,,,,,
-60864f46-e903-44b2-a59a-dd25d41eafbd,,Pediatric Ward,,Bahmni Hospital,,Login Location; Visit Location; Admission Location,,,,,,,,,,,
-43cf8857-0989-4fa0-a11e-0b2617129b3d,,Private Ward,,Bahmni Hospital,,Login Location; Visit Location; Admission Location,,,,,,,,,,,
-acf4dd0d-f665-43cf-a015-ab77165c8697,,AC Rooms,,Private Ward,,Admission Location,,,,,,,,,,,
-6e3caa21-20d1-4a1a-b905-8cde6b2e0a4b,,Non-AC Rooms,,Private Ward,,Admission Location,,,,,,,,,,,
-44cc2e40-ddce-415b-b5fc-2d99372ab117,,General OT,,Bahmni Hospital,,Operation Theater,,,,,,,,,,,
-9a1e5cd6-8e2e-41c6-a923-044cc55d7613,,Orthopaedic OT,,Bahmni Hospital,,Operation Theater,,,,,,,,,,,
-a36d6950-fbc7-4bd2-9369-a34dc763a1d8,,Gynaec OT,,Bahmni Hospital,,Operation Theater,,,,,,,,,,,
+Uuid,Void/Retire,Name,Description,Parent,Attribute|id_prefix,Tags,Address 1,Address 2,Address 3,Address 4,Address 5,Address 6,City/Village,County/District,State/Province,Postal Code,Country
+72636eba-29bf-4d6c-97c4-4b04d87a95b5,,Bahmni Hospital,,,,Facility,,,,,,,,,,,
+1973e910-160e-42ef-9daf-3f020f62c4cf,,Registration Desk,,Bahmni Hospital,,Login Location,,,,,,,,,,,
+5e232c47-8ff5-4c5c-8057-7e39a64fefa5,,OPD-1,,Bahmni Hospital,,Login Location; Appointment Location; Appointment Location,,,,,,,,,,,

```

Comment:
`Appointment Location` tag is added twice.

Reply:
removed.